lua/lspconfig/util.lua

```diff
@@ -438,8 +438,22 @@ function M.get_other_matching_providers(filetype)
   return other_matching_configs
 end
 
+function M.get_config_by_ft(filetype)
+  local configs = require 'lspconfig.configs'
+  local matching_configs = {}
+  for _, config in pairs(configs) do
+    local filetypes = config.filetypes or {}
+    for _, ft in pairs(filetypes) do
+      if ft == filetype then
+        table.insert(matching_configs, config)
+      end
+    end
+  end
+  return matching_configs
+end
+
 function M.get_active_client_by_name(bufnr, servername)
-  for _, client in pairs(vim.lsp.buf_get_clients(bufnr)) do
+  for _, client in pairs(vim.lsp.get_active_clients { bufnr = bufnr }) do
```

Comment:
`vim.lsp.get_active_clients()` was added in `0.8.0` and as I understand it nvim-lspconfig still supports `0.7.0`. Maybe it'd be best to defer these changes until `0.7.0` support is dropped?

Reply:
hmm i don't notice that it added in 0.8. use a variable with version check in there would better